src/components/Table/components/Row.tsx

```diff
@@ -176,13 +176,6 @@ function RowImpl<R extends Kinded, S>(props: RowProps<R>): ReactElement {
           const applyFirstContentColumnStyles = !isHeader && isFirstContentColumn;
           foundFirstContentColumn ||= applyFirstContentColumnStyles;
 
-          if (column.mw) {
```

Comment:
Makes a ton of sense to have this handled in the same place as the fr / size handling; great change!